coral-utils/src/scripts/Commons.js

```diff
@@ -124,7 +124,7 @@ class Commons {
     });
     
     const focusableElements = FOCUSABLE_ELEMENTS.slice();
-    this._focusableElementsSelector = focusableElements.join(',');
+    this._focusableElementsSelector = `${focusableElements.join(',')},[tabindex]`;
```

Comment:
Why is `[tabindex]` added ?

Reply:
The `FOCUSABLE_ELEMENTS_SELECTOR` should include `[tabindex=-1]` and `[tabindex=0]`. In the `CycleButton` use case, `coral-selectlist-item[tabindex="-1"]` elements in the `selectList` were being skipped when determining which element to focus first in the dropdown, and instead, the `button[coral-list-item]` items in the `actionList` received focus first, because they are actual `button` elements.